diff --git a/node_modules/@openeth/truffle-typings/index.d.ts b/node_modules/@openeth/truffle-typings/index.d.ts
index 5de91fb..9135d46 100644
--- a/node_modules/@openeth/truffle-typings/index.d.ts
+++ b/node_modules/@openeth/truffle-typings/index.d.ts
@@ -51,6 +51,7 @@ declare namespace Truffle {
     forwarder?: string
     useGSN?: boolean
     forceGasPrice?: string
+    factory?: string
   }
 
   export interface TransactionLog {
