diff --git a/node_modules/@openeth/truffle-typings/index.d.ts b/node_modules/@openeth/truffle-typings/index.d.ts
index 5de91fb..3d6a9f8 100644
--- a/node_modules/@openeth/truffle-typings/index.d.ts
+++ b/node_modules/@openeth/truffle-typings/index.d.ts
@@ -46,10 +46,10 @@ declare namespace Truffle {
     gas?: BN | number | string;
     gasPrice?: BN | number | string;
     value?: BN | string;
-    /* GSN-specific transaction details */
-    paymaster?: string
-    forwarder?: string
-    useGSN?: boolean
+    /* Enveloping-specific transaction details */
+    callVerifier?: string
+    callForwarder?: string
+    useEnveloping?: boolean
     forceGasPrice?: string
   }
 
