diff --git a/node_modules/@openeth/truffle-typings/index.d.ts b/node_modules/@openeth/truffle-typings/index.d.ts
index 5de91fb..7f48b5b 100644
--- a/node_modules/@openeth/truffle-typings/index.d.ts
+++ b/node_modules/@openeth/truffle-typings/index.d.ts
@@ -47,8 +47,8 @@ declare namespace Truffle {
     gasPrice?: BN | number | string;
     value?: BN | string;
     /* GSN-specific transaction details */
-    paymaster?: string
-    forwarder?: string
+    callVerifier?: string
+    callForwarder?: string
     useGSN?: boolean
     forceGasPrice?: string
   }
