diff --git a/node_modules/@openeth/truffle-typings/index.d.ts b/node_modules/@openeth/truffle-typings/index.d.ts
index 5de91fb..86d9264 100644
--- a/node_modules/@openeth/truffle-typings/index.d.ts
+++ b/node_modules/@openeth/truffle-typings/index.d.ts
@@ -46,11 +46,14 @@ declare namespace Truffle {
     gas?: BN | number | string;
     gasPrice?: BN | number | string;
     value?: BN | string;
-    /* GSN-specific transaction details */
-    paymaster?: string
-    forwarder?: string
-    useGSN?: boolean
+    /* Enveloping-specific transaction details */
+    callVerifier?: string
+    callForwarder?: string
+    useEnveloping?: boolean
     forceGasPrice?: string
+    tokenAmount?: BN | number | string;
+    tokenContract?: string;
+    forceGas?: BN | number | string;
   }
 
   export interface TransactionLog {
