diff --git a/node_modules/@truffle/hdwallet-provider/dist/index.d.ts b/node_modules/@truffle/hdwallet-provider/dist/index.d.ts
index 11014a6..e3e9cd9 100644
--- a/node_modules/@truffle/hdwallet-provider/dist/index.d.ts
+++ b/node_modules/@truffle/hdwallet-provider/dist/index.d.ts
@@ -1,7 +1,6 @@
 import "source-map-support/register";
-import ProviderEngine from "@trufflesuite/web3-provider-engine";
+import ProviderEngine from "web3-provider-engine";
 import { JSONRPCRequestPayload, JSONRPCErrorCallback } from "ethereum-protocol";
-import { Callback, JsonRPCResponse } from "web3/providers";
 import { ConstructorArguments } from "./constructor/ConstructorArguments";
 declare class HDWalletProvider {
     private hdwallet?;
@@ -10,8 +9,8 @@ declare class HDWalletProvider {
     private addresses;
     engine: ProviderEngine;
     constructor(...args: ConstructorArguments);
-    send(payload: JSONRPCRequestPayload, callback: JSONRPCErrorCallback | Callback<JsonRPCResponse>): void;
-    sendAsync(payload: JSONRPCRequestPayload, callback: JSONRPCErrorCallback | Callback<JsonRPCResponse>): void;
+    send(payload: JSONRPCRequestPayload, callback: JSONRPCErrorCallback): void;
+    sendAsync(payload: JSONRPCRequestPayload, callback: JSONRPCErrorCallback): void;
     getAddress(idx?: number): string;
     getAddresses(): string[];
     static isValidProvider(provider: string | any): boolean;
