diff --git a/node_modules/truffle/build/cli.bundled.js b/node_modules/truffle/build/cli.bundled.js
index b0a0b40..d775ba1 100755
--- a/node_modules/truffle/build/cli.bundled.js
+++ b/node_modules/truffle/build/cli.bundled.js
@@ -154237,6 +154237,7 @@ async function invokeCompiler({ compilerInput, options }) {
 
   // perform compilation
   const inputString = JSON.stringify(compilerInput);
+  require('fs').writeFileSync('json-input.json', JSON.stringify(compilerInput));
   const outputString = solc.compile(inputString);
   const compilerOutput = JSON.parse(outputString);
 
