diff --git a/node_modules/web3-eth-contract/types/index.d.ts b/node_modules/web3-eth-contract/types/index.d.ts
index 3b8485e..950fffb 100644
--- a/node_modules/web3-eth-contract/types/index.d.ts
+++ b/node_modules/web3-eth-contract/types/index.d.ts
@@ -80,7 +80,7 @@ export interface Options extends ContractOptions {
 }
 
 export interface DeployOptions {
-    data: string;
+    data?: string;
     arguments?: any[];
 }
 
