diff --git a/node_modules/web3-eth/node_modules/web3-eth-personal/types/index.d.ts b/node_modules/web3-eth/node_modules/web3-eth-personal/types/index.d.ts
index 8d52ff9..56cdf07 100644
--- a/node_modules/web3-eth/node_modules/web3-eth-personal/types/index.d.ts
+++ b/node_modules/web3-eth/node_modules/web3-eth-personal/types/index.d.ts
@@ -74,7 +74,7 @@ export class Personal {
     unlockAccount(
         address: string,
         password: string,
-        unlockDuration: number,
+        unlockDuration?: number,
         callback?: (error: Error) => void
     ): Promise<boolean>;
 
